### PR TITLE
Default features report bugfixes and updates

### LINKED ISSUE #47: Default features report bugfixes and updates
- Fix for `number_of_residential_units`.
- Fix for `maximum_roof_height_ft`.
- Fix for `footprint_area_sqft` (must divide by number of stories).
- Minor bug in program.rb related to `maximum_number_of_stories_above_ground`.
- For residential buildings, exclude unconditioned area from floor area.
- Determine residential buildings from building object.

### PR DESCRIPTION
**Pull request by joseph-robertson**
### Pull Request Description

[description here]

### Checklist (Delete lines that don't apply)

- [ ] Unit tests have been added or updated
- [ ] Documentation has been modified appropriately
- [x] All ci tests pass (green)
- [x] An [ISSUE](https://github.com/urbanopt/urbanopt-reporting-gem/issues) has been created that this is addressing. Issues will get added to the Change Log when the change_log.rb script is run.
- [x] This branch is up-to-date with develop

### CODE REVIEW

**Comment by joseph-robertson** [comment]
This PR has a dependency on https://github.com/urbanopt/urbanopt-example-geojson-project/pull/89, so I'm not sure we'd want to merge the entire thing at this point. Perhaps we can cherry-pick the non-residential related stuff though...

**Comment by kflemin** [comment]
@joseph-robertson let's just wait until everything is ready to merge this one. thanks!

**Review by kflemin — APPROVED** [review]

**Pull request merged**
